### ISSUE TITLE
Build output not compressed

### ISSUE THREAD

**Issue by ryanking13** [pyodide/pyodide#2038]
## 🐛 Bug

It looks like the build output of Pyodide is not compressed(uglified) now (after #2019?), which makes the output file size ~3X time bigger.

- Before: https://github.com/pyodide/pyodide/actions/runs/1541845126

```
5.2M Dec  5 18:32 pyodide.asm.data
2.0M Dec  5 18:32 pyodide.asm.js
 12M Dec  5 18:32 pyodide.asm.wasm
```

- After: https://github.com/pyodide/pyodide/actions/runs/1561467333

```
5.2M Dec 10 01:45 pyodide.asm.data
4.7M Dec 10 01:45 pyodide.asm.js
 38M Dec 10 01:45 pyodide.asm.wasm
```

**Comment by hoodmane** [comment]
Oops! This is just a flag in rollup.config.js I disabled for debugging on
fpcast branches.

**Comment by rth** [comment]
Could you recheck the benchmarks after the flag is removed? As maybe conclusions we had in the PR would also be impacted by this.

**Comment by hoodmane** [comment]
Okay I take it back I remembered to put that flag back so I don't know why this is happening.

**Comment by hoodmane** [comment]
On the last commit in #2019:
```
-rw-r--r-- 1 runner docker 5.2M Dec  9 18:43 pyodide.asm.data
-rw-r--r-- 1 runner docker 4.7M Dec  9 18:43 pyodide.asm.js
-rwxr-xr-x 1 runner docker  11M Dec  9 18:43 pyodide.asm.wasm
-rwxr-xr-x 1 runner docker  38M Dec  9 18:43 pyodide.asm.wasm.debug.wasm
```

**Comment by hoodmane** [comment]
Okay, specifically it's switching to `-g0` without `-gseparate-dwarf` that blows up the size. This was the last commit in:
https://github.com/pyodide/pyodide/pull/2026
You can see one commit before it's fine but with the last one it's much bigger.

**Comment by ryanking13** [comment]
Resolved by #2050, closing. We could later discuss debug flag somewhere else if needed.
